topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md

```diff
@@ -60,10 +70,21 @@ recordings:
   youtube_id: vnFQ2fR_fzw
   speakers:
   - wm75
+- youtube_id: -HQB1zyWWXM
+  length: 55M
+  galaxy_version: 24.1.2.dev0
```

Comment:
@wm75: Could you please check if this is correct? You only specified 24.1 which was closing an error. I added the most recent version, as your recording was not as long a go. :)